kbd 2.7 includes a rewrite of the setfont argument parser, with this rewritten
parser command-line option need to be passed **before** the font. Otherwise,
the parameters will be interpreted as font names. Without this patch, this
causes the service to fail starting with the error "Unable to find file: -C".

See: https://github.com/legionus/kbd/commit/aed486b760ccc7cdb0ae9a36b428bded6ea132c7

diff -upr openrc-0.55.1.orig/init.d/consolefont.in openrc-0.55.1/init.d/consolefont.in
--- openrc-0.55.1.orig/init.d/consolefont.in	2024-12-15 08:30:23.592468022 +0100
+++ openrc-0.55.1/init.d/consolefont.in	2024-12-15 08:30:59.756117156 +0100
@@ -52,7 +52,7 @@ start()
 	[ -d /dev/vc ] && ttydev=/dev/vc/
 	x=1
 	while [ $x -le $ttyn ]; do
-		if ! setfont $consolefont $param -C $ttydev$x >/dev/null; then
+		if ! setfont $param -C $ttydev$x $consolefont >/dev/null; then
 			retval=1
 			break
 		fi
